[BE]: Update flake8 to v6.1.0 and fix lints

Skylion007 opened the pull request:
Updates flake8 to v6.1.0 and fixes a few lints using sed and some ruff tooling.
- Replace `assert(0)` with `raise AssertionError()`
- Remove extraneous parenthesis i.e. 
  - `assert(a == b)` -> `assert a == b` 
  - `if(x > y or y < z):`->`if x > y or y < z:`
  - And `return('...')` -> `return '...'`

Skylion007 committed:
[BE]: Update flake8 to v6.1.0 and fix lints

Skylion007 commented:
@pytorchbot rebase

albanD commented on test/test_jit.py:
```diff
@@ -6169,7 +6169,7 @@ def test_not_cast(x):
             @torch.jit.script
             def test_mult(x, y):
-                return not(x, y)
+                return not (x, y)
```
Is this a not as well?

Skylion007 replied:
I don't follow what you mean here? This a non-functional change and is suppose to raise an error anyway.

malfet replied:
I wonder what that supposed to do? I.e. `not (x, y)` is always False, isn't it?

albanD commented on test/test_nn.py:
```diff
@@ -6865,7 +6865,7 @@ def test_linear_autograd(self, device, bias, weight_layout):
             module.weight = nn.Parameter(module.weight.to_sparse_coo())
         else:
-            assert(0)
+            assert (0)
```
Should this just be assert False?

albanD reviewed (approved):
Are the lint fixes related to the upgrade or just E275 errors? If they're unrelated, please make two independent PRs next time as we try to keep pin-upgrade PRs as small as possible.

Two small questions but the change sounds good otherwise

Skylion007 committed:
Reviewer comments

malfet commented on test/functorch/common_utils.py:
```diff
@@ -27,7 +27,7 @@ def loop(op, in_dims, out_dim, batch_size, *batched_args, **kwarg_values):
         flat_args, args_spec = pytree.tree_flatten(batched_args)
         flat_dims, dims_spec = pytree.tree_flatten(in_dims)
-        assert(args_spec == dims_spec)
+        assert (args_spec == dims_spec)
```
```suggestion
        assert args_spec == dims_spec
```

malfet commented on test/functorch/common_utils.py:
```diff
@@ -45,9 +45,9 @@ def loop2(op, in_dims1, in_dims2, out_dim1, out_dim2, batch_size1, batch_size2,
-    assert(args_spec == dims_spec1)
-    assert(args_spec == dims_spec2)
-    assert(len(flat_dims1) == len(flat_dims2))
+    assert (args_spec == dims_spec1)
+    assert (args_spec == dims_spec2)
+    assert (len(flat_dims1) == len(flat_dims2))
```
```suggestion
    assert args_spec == dims_spec1
    assert args_spec == dims_spec2
    assert len(flat_dims1) == len(flat_dims2)
```

malfet reviewed (approved):
LGTM, but don't we have a rule against extraneous parentheses? I.e. imo `assert(x==y)` should become `assert x==y` or indeed keep it as a function call

Skylion007 committed:
Apply suggestions from code review

Co-authored-by: Nikita Shulga <2453524+malfet@users.noreply.github.com>

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -599,13 +599,13 @@ def _lower_static_weighted_ref_module(
-        assert(isinstance(scale_node, Node))
-        assert(isinstance(zero_point_node, Node))
-        assert(issubclass(ref_class, nn.Module))
+        assert (isinstance(scale_node, Node))
+        assert (isinstance(zero_point_node, Node))
+        assert (issubclass(ref_class, nn.Module))
```
```suggestion
        assert isinstance(scale_node, Node)
        assert isinstance(zero_point_node, Node)
        assert issubclass(ref_class, nn.Module)
```

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -599,13 +599,13 @@ def _lower_static_weighted_ref_module(
         if q_node is None:
             continue
-        assert(ref_node is not None)
+        assert (ref_node is not None)
```
```suggestion
        assert ref_node is not None
```

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -557,7 +557,7 @@ def _match_static_pattern_with_two_inputs(
     q_node = node
     ref_node = q_node.args[0]
-    assert(isinstance(ref_node, Node))
+    assert (isinstance(ref_node, Node))
```
```suggestion
    assert isinstance(ref_node, Node)
```

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -487,14 +487,14 @@ def _match_static_pattern(
         relu_node = ref_node
         ref_node = relu_node.args[0]
-        assert(isinstance(ref_node, Node))
+        assert (isinstance(ref_node, Node))
```
```suggestion
        assert isinstance(ref_node, Node)
```

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -487,14 +487,14 @@ def _match_static_pattern(
     q_node = node
     ref_node = q_node.args[0]
-    assert(isinstance(ref_node, Node))
+    assert (isinstance(ref_node, Node))
```
```suggestion
    assert isinstance(ref_node, Node)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -712,7 +712,7 @@ def convert_eq_obs(
                 if weight_eq_obs_node is None:
                     return
-                assert(isinstance(modules[str(weight_eq_obs_node.target)], _WeightEqualizationObserver))
+                assert (isinstance(modules[str(weight_eq_obs_node.target)], _WeightEqualizationObserver))
```
```suggestion
                assert isinstance(modules[str(weight_eq_obs_node.target)], _WeightEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -696,7 +696,7 @@ def convert_eq_obs(
         elif weight_eq_obs_dict.get(node.name, None) is not None:
             weight_eq_obs = weight_eq_obs_dict.get(node.name)
-            assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+            assert (isinstance(weight_eq_obs, _WeightEqualizationObserver))
```
```suggestion
            assert isinstance(weight_eq_obs, _WeightEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -589,7 +589,7 @@ def update_obs_for_equalization(model: GraphModule, modules: Dict[str, nn.Module
                 if fused_module_supports_equalization(modules[str(op_node.target)]):
                     module = modules[str(op_node.target)][0]   # type: ignore[index]
-                    assert(nn_module_supports_equalization(module))
+                    assert (nn_module_supports_equalization(module))
```
```suggestion
                    assert nn_module_supports_equalization(module)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -578,7 +578,7 @@ def update_obs_for_equalization(model: GraphModule, modules: Dict[str, nn.Module
         if node.op == 'call_module' and isinstance(modules[node.target], _InputEqualizationObserver):
             input_eq_obs = modules[node.target]
-            assert(isinstance(input_eq_obs, _InputEqualizationObserver))
+            assert (isinstance(input_eq_obs, _InputEqualizationObserver))
```
```suggestion
            assert isinstance(input_eq_obs, _InputEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -546,10 +546,10 @@ def clear_weight_quant_obs_node(op_node: Node, modules: Dict[str, nn.Module]) ->
 
     weight_quant_obs = modules[str(weight_quant_obs_node.target)]
-    assert(isinstance(modules[str(weight_quant_obs_node.target)], ObserverBase))
+    assert (isinstance(modules[str(weight_quant_obs_node.target)], ObserverBase))
```
```suggestion
    assert isinstance(modules[str(weight_quant_obs_node.target)], ObserverBase)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -546,10 +546,10 @@ def clear_weight_quant_obs_node(op_node: Node, modules: Dict[str, nn.Module]) ->
     if weight_quant_obs_node is None:
         return
-    assert(isinstance(weight_quant_obs_node, Node))
+    assert (isinstance(weight_quant_obs_node, Node))
```
```suggestion
    assert isinstance(weight_quant_obs_node, Node)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -515,7 +515,7 @@ def scale_weight_functional(
 
     setattr(modules[weight_parent_name], weight_name, scaled_weight)
-    assert(torch.allclose(model.get_buffer(str(weight_node.target)), scaled_weight))
+    assert (torch.allclose(model.get_buffer(str(weight_node.target)), scaled_weight))
```
```suggestion
    assert torch.allclose(model.get_buffer(str(weight_node.target)), scaled_weight)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -487,14 +487,14 @@ def scale_weight_functional(
     if weight_node is None:
         return
-    assert(isinstance(weight_node, Node) and weight_node.op == 'get_attr')
+    assert (isinstance(weight_node, Node) and weight_node.op == 'get_attr')
```
```suggestion
    assert isinstance(weight_node, Node) and weight_node.op == 'get_attr'
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -453,7 +453,7 @@ def scale_weight_node(
     if bias is None:
         return
-    assert(isinstance(bias, torch.Tensor))
+    assert (isinstance(bias, torch.Tensor))
```
```suggestion
    assert isinstance(bias, torch.Tensor)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -426,12 +426,12 @@ def scale_weight_node(
     # If the following layer needs to be equalized then we will multiply its scale
     weight = op_module.weight
-    assert(isinstance(weight, torch.Tensor))
+    assert (isinstance(weight, torch.Tensor))
```
```suggestion
    assert isinstance(weight, torch.Tensor)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -426,12 +426,12 @@ def scale_weight_node(
     else:
         op_module = modules[str(node.target)]
-    assert(nn_module_supports_equalization(op_module) or custom_module_supports_equalization(op_module))
+    assert (nn_module_supports_equalization(op_module) or custom_module_supports_equalization(op_module))
```
```suggestion
    assert nn_module_supports_equalization(op_module) or custom_module_supports_equalization(op_module)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -389,10 +389,10 @@ def scale_input_observer(node: Node, modules: Dict[str, nn.Module]) -> None:
 
     input_quant_obs_node = node.args[0]
-    assert(isinstance(input_quant_obs_node, Node))
+    assert (isinstance(input_quant_obs_node, Node))
```
```suggestion
    assert isinstance(input_quant_obs_node, Node)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -389,10 +389,10 @@ def scale_input_observer(node: Node, modules: Dict[str, nn.Module]) -> None:
     """
     input_eq_obs = modules[str(node.target)]
-    assert(isinstance(input_eq_obs, _InputEqualizationObserver))
+    assert (isinstance(input_eq_obs, _InputEqualizationObserver))
```
```suggestion
    assert isinstance(input_eq_obs, _InputEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -364,7 +364,7 @@ def maybe_get_next_input_eq_obs(node: Node, modules: Dict[str, nn.Module]) -> Op
 
     maybe_eq_obs = modules[str(maybe_eq_obs_node)]
-    assert(isinstance(maybe_eq_obs, _InputEqualizationObserver))
+    assert (isinstance(maybe_eq_obs, _InputEqualizationObserver))
```
```suggestion
    assert isinstance(maybe_eq_obs, _InputEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -342,7 +342,7 @@ def maybe_get_next_input_eq_obs(node: Node, modules: Dict[str, nn.Module]) -> Op
     """
 
-    assert(node_supports_equalization(node, modules))
+    assert (node_supports_equalization(node, modules))
```
```suggestion
    assert node_supports_equalization(node, modules)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -291,35 +291,35 @@ def get_op_node_and_weight_eq_obs(
     """ Gets the weight equalization observer node if it exists.
     """
-    assert(op_node.op == 'call_function')
+    assert (op_node.op == 'call_function')
```
```suggestion
    assert op_node.op == 'call_function'
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -291,35 +291,35 @@ def get_op_node_and_weight_eq_obs(
         if weight_node is not None:
             weight_eq_obs = modules[str(weight_node.target)]
-            assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+            assert (isinstance(weight_eq_obs, _WeightEqualizationObserver))
```
```suggestion
            assert isinstance(weight_eq_obs, _WeightEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -291,35 +291,35 @@ def get_op_node_and_weight_eq_obs(
         weight_eq_obs = equalization_node_name_to_qconfig.get(op_node.name, None).weight()
 
-        assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+        assert (isinstance(weight_eq_obs, _WeightEqualizationObserver))
```
```suggestion
        assert isinstance(weight_eq_obs, _WeightEqualizationObserver)
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -291,35 +291,35 @@ def get_op_node_and_weight_eq_obs(
         assert maybe_equalization_node_name_to_config is not None
         equalization_node_name_to_qconfig: Dict[str, Any] = maybe_equalization_node_name_to_config  # type: ignore[assignment]
-        assert(equalization_node_name_to_qconfig.get(op_node.name, None) is not None)
+        assert (equalization_node_name_to_qconfig.get(op_node.name, None) is not None)
```
```suggestion
        assert equalization_node_name_to_qconfig.get(op_node.name, None) is not None
```

malfet commented on torch/ao/quantization/fx/_equalize.py:
```diff
@@ -291,35 +291,35 @@ def get_op_node_and_weight_eq_obs(
             break
 
-    assert(op_node is not None)
+    assert (op_node is not None)
```
```suggestion
    assert op_node is not None
```

malfet commented on torch/ao/quantization/fuser_method_mappings.py:
```diff
@@ -147,7 +147,7 @@ def fuse_convtranspose_bn(is_qat, convt, bn):
         >>> m2 = fuse_convtranspose_bn(m1, b1)
     """
-    assert(convt.training == bn.training),\
+    assert (convt.training == bn.training), \
```
```suggestion
    assert convt.training == bn.training, \
```

malfet commented on torch/ao/quantization/fuser_method_mappings.py:
```diff
@@ -118,14 +118,14 @@ def fuse_linear_bn(is_qat, linear, bn):
         >>> m2 = fuse_linear_bn(m1, b1)
     """
-    assert(linear.training == bn.training),\
+    assert (linear.training == bn.training), \
```
```suggestion
    assert linear.training == bn.training, \
```

malfet commented on torch/ao/quantization/fuser_method_mappings.py:
```diff
@@ -71,7 +71,7 @@ def fuse_conv_bn_relu(is_qat, conv, bn, relu):
         >>> m2 = fuse_conv_bn_relu(m1, b1, r1)
     """
-    assert(conv.training == bn.training == relu.training),\
+    assert (conv.training == bn.training == relu.training), \
```
```suggestion
    assert conv.training == bn.training == relu.training, \
```

malfet commented on torch/ao/quantization/fuser_method_mappings.py:
```diff
@@ -31,7 +31,7 @@ def fuse_conv_bn(is_qat, conv, bn):
         >>> m2 = fuse_conv_bn(m1, b1)
     """
-    assert(conv.training == bn.training),\
+    assert (conv.training == bn.training), \
```
```suggestion
    assert conv.training == bn.training, \
```

malfet commented on torch/ao/quantization/experimental/observer.py:
```diff
@@ -58,8 +58,8 @@ def _calculate_qparams(self, signed: bool, min_val=None, max_val=None):
         # check for valid inputs of b, k
-        assert(self.k and self.k != 0)
-        assert(self.b % self.k == 0)
+        assert (self.k and self.k != 0)
+        assert (self.b % self.k == 0)
```
```suggestion
        assert self.k and self.k != 0
        assert self.b % self.k == 0
```

malfet commented on torch/ao/quantization/backend_config/onednn.py:
```diff
@@ -85,7 +85,7 @@ def _fuse_linear_bn_leaky_relu(is_qat, linear, bn, leaky_relu):
         >>> m2 = _fuse_linear_bn_leaky_relu(m1, b1, lr)
     """
-    assert(linear.training == bn.training and bn.training == leaky_relu.training),\
+    assert (linear.training == bn.training and bn.training == leaky_relu.training), \
```
```suggestion
    assert linear.training == bn.training and bn.training == leaky_relu.training, \
```

malfet commented on test/test_mps.py:
```diff
@@ -4157,7 +4157,7 @@ def test_cumprod_minus_one_axis(self):
             # Test with axis -1
             cpu_x = None
-            if(dtype == torch.float32):
+            if (dtype == torch.float32):
```
```suggestion
            if dtype == torch.float32:
```

malfet commented on test/test_mps.py:
```diff
@@ -4122,7 +4122,7 @@ def test_cumsum_minus_one_axis(self):
             # Test with axis -1
             cpu_x = None
-            if(dtype == torch.float32):
+            if (dtype == torch.float32):
```
```suggestion
            if dtype == torch.float32:
```

malfet commented on test/test_jit.py:
```diff
@@ -12837,7 +12837,7 @@ def test_file_format_serialization(self):
         for i, offset in enumerate(parsed_serialized_offsets):
             data = reader.get_record(str(offset))
-            assert(data == buffers[i])
+            assert (data == buffers[i])
```
```suggestion
            assert data == buffers[i]
```

malfet commented on test/test_jit.py:
```diff
@@ -8595,7 +8595,7 @@ def func():
                 for tensor_type in dtypes:
                     # a couple of ops aren't implemented for non-floating types
-                    if(not tensor_type.is_floating_point or (dtype is not None and not dtype.is_floating_point)):
+                    if (not tensor_type.is_floating_point or (dtype is not None and not dtype.is_floating_point)):
```
```suggestion
                    if not tensor_type.is_floating_point or (dtype is not None and not dtype.is_floating_point):
```

malfet commented on test/test_jit.py:
```diff
@@ -8557,15 +8557,15 @@ def test_check_not():
+        if (dtype == torch.int32):
             return "Int"
-        if(dtype == torch.bool):
+        if (dtype == torch.bool):
```
```suggestion
        if dtype == torch.bool:
```

malfet commented on test/test_jit.py:
```diff
@@ -8557,15 +8557,15 @@ def test_check_not():
+        if (dtype == torch.int64):
             return "Long"
-        if(dtype == torch.int32):
+        if (dtype == torch.int32):
```
```suggestion
        if dtype == torch.int32:
```

malfet commented on test/test_jit.py:
```diff
@@ -8557,15 +8557,15 @@ def test_check_not():
+        if (dtype == torch.float64):
             return "Double"
-        if(dtype == torch.int64):
+        if (dtype == torch.int64):
```
```suggestion
        if dtype == torch.int64:
```

malfet commented on test/test_jit.py:
```diff
@@ -8557,15 +8557,15 @@ def test_check_not():
+        if (dtype == torch.float32):
             return "Float"
-        if(dtype == torch.float64):
+        if (dtype == torch.float64):
```
```suggestion
        if dtype == torch.float64:
```

malfet commented on test/test_jit.py:
```diff
@@ -8557,15 +8557,15 @@ def test_check_not():
 
     def _dtype_to_jit_name(self, dtype):
-        if(dtype == torch.float32):
+        if (dtype == torch.float32):
```
```suggestion
        if dtype == torch.float32:
```

malfet commented on test/test_jit.py:
```diff
@@ -5746,7 +5746,7 @@ def fn(x):
         graph = _propagate_shapes(fn.graph, (x,), False)
         default = torch.get_default_dtype()
-        if(default == torch.float):
+        if (default == torch.float):
```
```suggestion
        if default == torch.float:
```

malfet commented on test/test_jit.py:
```diff
@@ -1875,8 +1875,8 @@ def t(x, p: float, t: bool):
                         o_ref.sum().backward()
-                        assert(o.equal(o_ref))
-                        assert(x.grad.equal(x_ref.grad))
+                        assert (o.equal(o_ref))
+                        assert (x.grad.equal(x_ref.grad))
```
```suggestion
                        assert o.equal(o_ref)
                        assert x.grad.equal(x_ref.grad)
```

malfet commented on test/test_fx.py:
```diff
@@ -3772,7 +3772,7 @@ def additional_globals(self):
 
             def process_inputs(self, *inputs):
-                assert(len(inputs) == 1)
+                assert (len(inputs) == 1)
```
```suggestion
                assert len(inputs) == 1
```

malfet commented on test/test_fx.py:
```diff
@@ -3743,7 +3743,7 @@ def additional_globals(self):
 
             def process_inputs(self, *inputs):
-                assert(len(inputs) == 1)
+                assert (len(inputs) == 1)
```
```suggestion
                assert len(inputs) == 1
```

malfet committed:
Apply suggestions from code review

- Remove extraneous parentheses
- Replace `assert False` with `raise AssertionError()`

malfet commented on test/jit/test_scriptmod_ann.py:
```diff
@@ -129,7 +129,7 @@ def forward(self, y: List[int]):
         with warnings.catch_warnings(record=True) as w:
             self.checkModule(M(), ([1, 2, 3],))
-        assert(len(w) == 0)
+        assert (len(w) == 0)
```
```suggestion
        assert len(w) == 0
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -147,7 +147,7 @@ class testVariousModelVersions(TestCase):
         def check_model_version(model_path, expect_version):
             actual_version = _get_model_bytecode_version(model_path)
-            assert(actual_version == expect_version)
+            assert (actual_version == expect_version)
```
```suggestion
            assert actual_version == expect_version
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -174,7 +174,7 @@ def test_bytecode_values_for_all_backport_functions(self):
                 current_to_version = current_from_version - 1
                 backport_success = _backport_for_mobile(input_model_path, tmp_output_model_path_backport, current_to_version)
-                assert(backport_success)
+                assert (backport_success)
```
```suggestion
                assert backport_success
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -187,7 +187,7 @@ def test_bytecode_values_for_all_backport_functions(self):
                 expect_result_clean = "".join(expect_bytecode_pkl.split())
                 isMatch = fnmatch.fnmatch(acutal_result_clean, expect_result_clean)
-                assert(isMatch)
+                assert (isMatch)
```
```suggestion
                assert isMatch
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -254,7 +254,7 @@ def test_backport_bytecode_from_file_to_file(self):
                     tmp_backport_model_path,
                     maximum_checked_in_model_version - 1)
-                assert(success)
+                assert (success)
```
```suggestion
                assert success
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -266,7 +266,7 @@ def test_backport_bytecode_from_file_to_file(self):
                 expect_result_clean = "".join(expected_result.split())
                 isMatch = fnmatch.fnmatch(acutal_result_clean, expect_result_clean)
-                assert(isMatch)
+                assert (isMatch)
```
```suggestion
                assert isMatch
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -291,7 +291,7 @@ def test_backport_bytecode_from_file_to_buffer(self):
             bytesio = io.BytesIO(script_module_v4_buffer)
             backport_version = _get_model_bytecode_version(bytesio)
-            assert(backport_version == maximum_checked_in_model_version - 1)
+            assert (backport_version == maximum_checked_in_model_version - 1)
```
```suggestion
            assert backport_version == maximum_checked_in_model_version - 1
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -306,8 +306,8 @@ def test_get_model_ops_and_info(self):
         ops_v6 = _get_model_ops_and_info(script_module_v6)
-        assert(ops_v6["aten::add.int"].num_schema_args == 2)
-        assert(ops_v6["aten::add.Scalar"].num_schema_args == 2)
+        assert (ops_v6["aten::add.int"].num_schema_args == 2)
+        assert (ops_v6["aten::add.Scalar"].num_schema_args == 2)
```
```suggestion
        assert ops_v6["aten::add.int"].num_schema_args == 2
        assert ops_v6["aten::add.Scalar"].num_schema_args == 2
```

malfet commented on test/mobile/test_bytecode.py:
```diff
@@ -322,7 +322,7 @@ def forward(self, x):
         buffer.seek(0)
         type_list = _get_mobile_model_contained_types(buffer)
-        assert(len(type_list) >= 0)
+        assert (len(type_list) >= 0)
```
```suggestion
        assert len(type_list) >= 0
```

malfet commented on test/mobile/test_lite_script_module.py:
```diff
@@ -82,7 +82,7 @@ def forward(self, x, y, z):
             buffer.seek(0)
 
-            assert(b"callstack_debug_map.pkl" in exported_module)
+            assert (b"callstack_debug_map.pkl" in exported_module)
```
```suggestion
            assert b"callstack_debug_map.pkl" in exported_module
```

malfet commented on test/quantization/core/test_quantized_op.py:
```diff
@@ -4101,7 +4101,7 @@ def test_qlinear_unpack(self, W, use_channelwise):
     @override_qengines
     def test_qlinear_qnnpack_free_memory_and_unpack(self, W):
-        assert(qengine_is_qnnpack)
+        assert (qengine_is_qnnpack)
```
```suggestion
        assert qengine_is_qnnpack
```

malfet commented on test/quantization/core/test_workflow_ops.py:
```diff
@@ -879,7 +879,7 @@ def _test_backward_per_channel_cachemask_impl(self, device):
             np.testing.assert_allclose(
                 dX.cpu().detach().numpy(), X.grad.cpu().detach().numpy(), rtol=tolerance, atol=tolerance)
-            assert(X.grad.dtype == float_type)
+            assert (X.grad.dtype == float_type)
```
```suggestion
            assert X.grad.dtype == float_type
```

malfet commented on test/test_autograd.py:
```diff
@@ -4319,7 +4319,7 @@ def get_id():
         for thread, ranges in threads:
             for range in ranges:
-                assert(len(range) == 3)
+                assert (len(range) == 3)
```
```suggestion
                assert len(range) == 3
```

malfet commented on test/test_autograd.py:
```diff
@@ -4340,7 +4340,7 @@ def get_id():
             return [child.id for child in event.cpu_children]
 
-        assert([get_children_ids(event) for event in events] == res)
+        assert ([get_children_ids(event) for event in events] == res)
```
```suggestion
        assert [get_children_ids(event) for event in events] == res
```

malfet commented on test/test_bundled_images.py:
```diff
@@ -30,7 +30,7 @@ def save_and_load(sm):
     # turn NCHW to HWC
     if img_tensor.dim() == 4:
-        assert(img_tensor.size(0) == 1)
+        assert (img_tensor.size(0) == 1)
```
```suggestion
        assert img_tensor.size(0) == 1
```

malfet commented on test/test_cuda.py:
```diff
@@ -1388,7 +1388,7 @@ def test_grad_scale_will_not_overflow(self):
         scaler.step(optimizer)
         scaler.update()
-        assert(scaler._scale != float('inf') and scaler._scale != float('nan'))
+        assert (scaler._scale != float('inf') and scaler._scale != float('nan'))
```
```suggestion
        assert scaler._scale != float('inf') and scaler._scale != float('nan')
```

malfet commented on test/test_fx.py:
```diff
@@ -1676,8 +1676,8 @@ def forward(self, x):
 
-        assert(all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
-                   for node in traced.graph.nodes))
+        assert (all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
+                    for node in traced.graph.nodes))
```
```suggestion
        assert all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
                   for node in traced.graph.nodes)
```

malfet commented on test/test_fx.py:
```diff
@@ -1733,8 +1733,8 @@ def forward(self, x):
         shape_prop.ShapeProp(traced_3d).propagate(x_3d)
-        assert(all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
-                   for node in traced_3d.graph.nodes))
+        assert (all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
+                    for node in traced_3d.graph.nodes))
```
```suggestion
        assert all(node.meta['tensor_meta'].memory_format is torch.contiguous_format
                   for node in traced_3d.graph.nodes)
```

malfet commented on test/test_fx.py:
```diff
@@ -2548,7 +2548,7 @@ def forward(self, x):
 
         traced = torch.fx.symbolic_trace(Foo())
-        assert(all('constant' not in node.target for node in traced.graph.nodes))
+        assert (all('constant' not in node.target for node in traced.graph.nodes))
```
```suggestion
        assert all('constant' not in node.target for node in traced.graph.nodes)
```

malfet commented on test/test_fx.py:
```diff
@@ -2824,7 +2824,7 @@ def forward(self, x, y):
         self.assertEqual(mod_true(3, True), 6)
         print(mod_true.code)
-        assert(any(i.target == torch._assert for i in mod_true.graph.nodes))
+        assert (any(i.target == torch._assert for i in mod_true.graph.nodes))
```
```suggestion
        assert any(i.target == torch._assert for i in mod_true.graph.nodes)
```

malfet commented on test/test_fx.py:
```diff
@@ -3607,17 +3607,17 @@ def verify_pytree(f, inp):
 
             assert num_flat_args == 0 or "tree_flatten_spec" in nf.code
-            assert(sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args)
+            assert (sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args)
```
```suggestion
            assert sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args
```

malfet commented on test/test_fx.py:
```diff
@@ -3607,17 +3607,17 @@ def verify_pytree(f, inp):
             self.assertEqual(nf(val), orig_out)
             assert "tree_flatten_spec" not in nf.code
-            assert(sum([i.op == 'placeholder' for i in nf.graph.nodes]) == 1)
+            assert (sum([i.op == 'placeholder' for i in nf.graph.nodes]) == 1)
```
```suggestion
            assert sum([i.op == 'placeholder' for i in nf.graph.nodes]) == 1
```

malfet commented on test/test_fx.py:
```diff
@@ -3607,17 +3607,17 @@ def verify_pytree(f, inp):
             self.assertEqual(nf(val), orig_out)
             assert num_flat_args == 0 or "tree_flatten_spec" in nf.code
-            assert(sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args)
+            assert (sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args)
```
```suggestion
            assert sum([i.op == 'placeholder' for i in nf.graph.nodes]) == num_flat_args
```

malfet commented on test/test_fx.py:
```diff
@@ -3708,7 +3708,7 @@ def additional_globals(self):
 
             def process_inputs(self, *inputs):
-                assert(len(inputs) == 1)
+                assert (len(inputs) == 1)
```
```suggestion
                assert len(inputs) == 1
```

malfet commented on test/test_overrides.py:
```diff
@@ -217,7 +217,7 @@ class SubTensor(torch.Tensor):
     @classmethod
     def __torch_function__(cls, func, types, args=(), kwargs=None):
-        if(kwargs is None):
+        if (kwargs is None):
```
```suggestion
        if kwargs is None:
```

malfet commented on test/test_overrides.py:
```diff
@@ -368,7 +368,7 @@ class TensorLike:
     @classmethod
     def __torch_function__(cls, func, types, args=(), kwargs=None):
-        if(kwargs is None):
+        if (kwargs is None):
```
```suggestion
        if kwargs is None:
```

malfet commented on test/test_reductions.py:
```diff
@@ -1353,7 +1353,7 @@ def check_sum_all(tensor: torch.Tensor) -> None:
                                             memory_format, compare_data=True, default_is_preserve=False):
 
-        assert(memory_format == torch.channels_last or memory_format == torch.channels_last_3d)
+        assert (memory_format == torch.channels_last or memory_format == torch.channels_last_3d)
```
```suggestion
        assert memory_format == torch.channels_last or memory_format == torch.channels_last_3d
```

malfet commented on test/test_torch.py:
```diff
@@ -5210,7 +5210,7 @@ def test_multinomial_rng_state_advance(self, device, dtype):
                                             memory_format, compare_data=True, default_is_preserve=False):
 
-        assert(memory_format == torch.channels_last or memory_format == torch.channels_last_3d)
+        assert (memory_format == torch.channels_last or memory_format == torch.channels_last_3d)
```
```suggestion
        assert memory_format == torch.channels_last or memory_format == torch.channels_last_3d
```

malfet commented on torch/__init__.py:
```diff
@@ -191,7 +191,7 @@ def _load_global_deps() -> None:
             'nvtx': 'libnvToolsExt.so.*[0-9]',
         }
-        is_cuda_lib_err = [lib for lib in cuda_libs.values() if(lib.split('.')[0] in err.args[0])]
+        is_cuda_lib_err = [lib for lib in cuda_libs.values() if (lib.split('.')[0] in err.args[0])]
```
```suggestion
        is_cuda_lib_err = [lib for lib in cuda_libs.values() if lib.split('.')[0] in err.args[0]]
```

malfet commented on torch/_functorch/benchmark_utils.py:
```diff
@@ -75,7 +75,7 @@ def is_gpu_compute_event(event):
     sorted_gpu_events = []
     for event in events:
-        if(not is_gpu_compute_event(event)):
+        if (not is_gpu_compute_event(event)):
```
```suggestion
        if not is_gpu_compute_event(event):
```

malfet commented on torch/_functorch/benchmark_utils.py:
```diff
@@ -102,7 +102,7 @@ def is_mm_conv_event(event):
     sorted_events = []
     for event in gpu_events:
-        if(not is_mm_conv_event(event)):
+        if (not is_mm_conv_event(event)):
```
```suggestion
        if not is_mm_conv_event(event):
```

malfet commented on torch/ao/nn/sparse/quantized/utils.py:
```diff
@@ -22,7 +22,7 @@ class LinearBlockSparsePattern:
 
     def __init__(self, row_block_size=1, col_block_size=4):
-        assert(_is_valid_linear_block_sparse_pattern(row_block_size, col_block_size))
+        assert (_is_valid_linear_block_sparse_pattern(row_block_size, col_block_size))
```
```suggestion
        assert _is_valid_linear_block_sparse_pattern(row_block_size, col_block_size)
```

malfet commented on torch/ao/quantization/fx/_lower_to_native_backend.py:
```diff
@@ -778,14 +778,14 @@ def _lower_static_weighted_ref_functional(
+        assert isinstance(weight_dq_node, Node)
         quantized_weight = weight_dq_node.args[0]
-        assert(isinstance(quantized_weight, Node))
+        assert (isinstance(quantized_weight, Node))
```
```suggestion
        assert isinstance(quantized_weight, Node)
```

malfet commented on torch/ao/quantization/fx/prepare.py:
```diff
@@ -1585,7 +1585,7 @@ def insert_observers_for_model(
                             # modules, not just for LSTM.
                             _insert_dequant_stubs_for_custom_module_lstm_output(node, model, named_modules, model.graph)
-                            if(node.target not in custom_module_names_already_swapped):
+                            if (node.target not in custom_module_names_already_swapped):
```
```suggestion
                            if node.target not in custom_module_names_already_swapped:
```

malfet commented on torch/ao/quantization/fx/prepare.py:
```diff
@@ -1627,7 +1627,7 @@ def insert_observers_for_model(
 
                                 if qhandler is not None and qhandler.is_custom_module():
-                                    if(node.target not in custom_module_names_already_swapped):
+                                    if (node.target not in custom_module_names_already_swapped):
```
```suggestion
                                    if node.target not in custom_module_names_already_swapped:
```

malfet commented on torch/fx/experimental/optimization.py:
```diff
@@ -220,7 +220,7 @@ def find(self, v: int) -> int:
         if v == par:
             return v
-        assert(par is not None)
+        assert (par is not None)
```
```suggestion
        assert par is not None
```

malfet commented on torch/fx/experimental/optimization.py:
```diff
@@ -360,14 +360,14 @@ def get_color(n):
             uf.make_set(cur_idx)
         elif node.op == 'call_method' and node.target == 'to_dense':
-            assert(get_color(node.args[0]) is not None)
+            assert (get_color(node.args[0]) is not None)
```
```suggestion
            assert get_color(node.args[0]) is not None
```

malfet commented on torch/fx/experimental/symbolic_shapes.py:
```diff
@@ -1359,7 +1359,7 @@ def solve(self, disable_congruences=True, disable_equivalences=True):
         # as long as there are symbols with equalities, solve for them
         # NOTE(avik): this is guaranteed to terminate (#iterations <= #symbols)
-        while(self._symbols_with_equalities):
+        while (self._symbols_with_equalities):
```
```suggestion
        while self._symbols_with_equalities:
```

malfet commented on torch/fx/graph.py:
```diff
@@ -670,7 +670,7 @@ def process_outputs(self, out: Any) -> Any:
         if not isinstance(out, (list, tuple)):
             out = [out]
-        assert(self.pytree_info.out_spec is not None)
+        assert (self.pytree_info.out_spec is not None)
```
```suggestion
        assert self.pytree_info.out_spec is not None
```

malfet commented on torch/fx/node.py:
```diff
@@ -82,7 +82,7 @@ def _type_repr(obj):
         return f'{obj.__module__}.{obj.__qualname__}'
     if obj is ...:
-        return('...')
+        return ('...')
```
```suggestion
        return '...'
```

malfet commented on torch/fx/passes/net_min_base.py:
```diff
@@ -698,7 +698,7 @@ def minimize(
             return self._accumulate_traverse(nodes)
 
-        if(self.settings.traverse_method == "skip"):
+        if (self.settings.traverse_method == "skip"):
```
```suggestion
        if self.settings.traverse_method == "skip":
```

malfet commented on torch/testing/_internal/jit_metaprogramming_utils.py:
```diff
@@ -331,7 +331,7 @@ def value_to_literal(value):
     kwargs_str = ', '.join([k + '=' + value_to_literal(v) for k, v in kwargs.items()])
     self_arg = args[0]
-    if(func_type == 'method'):
+    if (func_type == 'method'):
```
```suggestion
    if func_type == 'method':
```

malfet committed:
Apply suggestions from code review

malfet commented:
@pytorchbot merge
